tests/drivers/watchdog/wdt_error_cases/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&uart135 {
+    pinctrl-0 = < &uart136_default >;
+    pinctrl-1 = < &uart136_sleep >;
+};
```

Comment:
Why?

Reply:
I'm not 100% sure this should be here. I add this because:
- I run Twister on App and Radio using the same UART port (on PC side); So it was easier for me to have both cores printing on the same GPIOs. It's enought to run twister twice, each run on other cpu with different UART port. 
- I used secdom FW with STM traces; it prints logs on GPIOs defined in pincontrol for uart135.

I will remove this to have all tests on Radio core printing on the same UART.